modules/contact/src/main/kotlin/edu/stanford/spezi/modules/contact/model/PersonNameComponents.kt

```diff
@@ -0,0 +1,22 @@
+package edu.stanford.spezi.modules.contact.model
+
+data class PersonNameComponents(
+    val namePrefix: String? = null,
+    val givenName: String? = null,
+    val middleName: String? = null,
+    val familyName: String? = null,
+    val nameSuffix: String? = null,
+    val nickname: String? = null,
+) {
+    fun formatted(): String {
```

Comment:
Similary as the address formatted extension function, we can mark this with composable and use this as a key for remember

Reply:
see other comment